airbyte-integrations/connectors/source-harvest/acceptance-test-config.yml

```diff
@@ -0,0 +1,26 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector/source-acceptance-tests.md)
+# for more information about how to configure these tests
```

Comment:
I'm not sure if this comment should be left, it exists only to start developing the connector, WDYT?